src/servers/ZoneServer2016/zoneserver.ts

```diff
@@ -5433,6 +5433,10 @@ export class ZoneServer2016 extends EventEmitter {
     let durability: number = 2000;
     switch (true) {
       case this.isWeapon(itemDefinitionId):
+        if (itemDefinitionId == Items.WEAPON_CROWBAR) {
+          durability = Math.floor(Math.random() * 2001);
```

Comment:
Why 2001 ?

Reply:
So the value 5400 or 2000 can be generated from the random number generator, but perhaps just doing 2000 and 5400 is the same thing?

Reply:
It's true that 1 is excluded so 0.999999999 * 2000 would be the max number but since you do Math.floor 1999.9999 would be 2000. Right now it could result in a crowbar having 2001 durability

Reply:
I see. I updated the code to be reflective of 5400 and 2000 :D